app/src/main/java/com/example/vybe/vibefactory/AngryVibe.java

```diff
@@ -0,0 +1,35 @@
+package com.example.vybe.vibefactory;
+
+import com.example.vybe.R;
+
+public class AngryVibe implements Vibe {
+    private int color;
+    private int emoticon;
+
+    /**
+     * This Constructor sets the new vibe's color and emoticon
+     */
+    public AngryVibe() {
+        this.color = R.color.Red;
+        this.emoticon = R.drawable.ic_angry;
+    }
+
+    /**
+     * This gets the string value of the Vibe class selected by a user
+     * @return
+     * The string that describes the Vibe selected
+     */
+    public String getVibe() {
+        return "Angry";
```

Comment:
Seems like you have lower-case string names everywhere in the code. Does it matter if these ones aren't in lower-case?

Reply:
According to @jacobwong98, it should not matter - but just changing these to lowercase anyways for consistency